skimage/transform/radon_transform.py

```diff
@@ -1,10 +1,19 @@
 import numpy as np
-from scipy.fftpack import fft, ifft, fftfreq
+
 from scipy.interpolate import interp1d
 from ._warps_cy import _warp_fast
 from ._radon_transform import sart_projection_update
+from .._shared.fft import fftmodule
 from warnings import warn
 
+if fftmodule is np.fft:
+    # fallback from scipy.fft to scipy.fftpack instead of numpy.fft
+    # (fftpack preserves single precision while numpy.fft does not)
+    from scipy.fftpack import fft, ifft
```

Comment:
why not always fallback to this?

Reply:
1.) NumPy's pypocketfft will be MUCH faster than old `scipy.fftpack`, at least for non-favorable sizes (e.g. not factors of 2, 3, 5, 7 etc).
2.) `scipy.fft` follows numpy's API much more closely than `scipy.fftpack`. e.g. `fftpack`'s `rfft`/`irfft` do not organize data in the same way as NumPy's so the places in other files that were using those cannot just switch to `fftpack` as a drop-in replacement for the existing code.

Reply:
For `radon`/`iradon` I did not look closely enough to see if single-precision is actually being used. If everything in this file gets promoted to `float64` anyways then we probably don't need to special case here and can just fallback to `numpy.fft` as done elsewhere.

Reply:
man, i feel you pain in making this PR. well thought out. Thanks for the explanation. 

I wish we could just bumpt all the way up to scipy 1.4 for you ;)